pkgs/top-level/python-packages.nix

```diff
@@ -16568,6 +16572,30 @@ in {
     };
   };
 
+  pyobjc = if stdenv.isDarwin
+    then callPackage ../development/python-modules/pyobjc {}
+    else throw "pyobjc can only be built on Mac OS";
+
+  pyodbc = buildPythonPackage rec {
```

Comment:
I think your rebase got mixed up. There is already a
```
pyodbc = callPackage ../development/python-modules/pyodbc { };
```
below. This results in an evaluation error:
```
error: attribute ‘pyodbc’ at /home/rycee/devel/nixpkgs/pkgs/top-level/python-packages.nix:16599:3 already defined at /home/rycee/devel/nixpkgs/pkgs/top-level/python-packages.nix:16579:3
```